dbhydra/dbhydra_core.py

```diff
@@ -1287,17 +1284,41 @@ def get_all_columns(self):
 
     def get_all_types(self):
 
+        only_types, lengths = self.get_types_and_lengths()
+        for i in range(len(only_types)):
+            if lengths[i] is not None:
+                only_types[i] = only_types[i] + f"({lengths[i]})"
+        return (only_types)
+
+    def get_types_and_lengths(self):
         information_schema_table = Table(self.db1, 'INFORMATION_SCHEMA.COLUMNS', ['DATA_TYPE'], ['nvarchar(50)'])
         query = f"SELECT DATA_TYPE,character_maximum_length FROM INFORMATION_SCHEMA.COLUMNS WHERE TABLE_SCHEMA = '{self.db1.DB_DATABASE}' AND TABLE_NAME  = '" + self.name + "'"
         types = information_schema_table.select(query)
         only_types = [x[0] for x in types]
         lengths = [x[1] for x in types]
 
-        for i in range(len(only_types)):
-            if lengths[i] is not None:
-                only_types[i] = only_types[i] + f"({lengths[i]})"
-        return (only_types)
+        return only_types,lengths
+
+    def get_converted_python_types(self):
+        SQL_TO_PYTHON = {v: k for k, v in PYTHON_TO_MYSQL_DATA_MAPPING.items()}
+        python_types = []
+        for type in self.types:
+            if "varchar" in type:
+                python_types.append("str")
+            elif "nvarchar" in type:
+                python_types.append("str")
```

Comment:
`if "varchar" in type or "nvarchar" in type:`

Reply:
if `nvarchar` is in type then the row 1308 is never executed, because the `varchar` in type is always True

--> if "varchar" in type is sufficient